test/ExecutionFramework.h

```diff
@@ -287,7 +282,7 @@ class ExecutionFramework
 	bool m_transactionSuccessful = true;
 	Address m_sender = account(0);
 	Address m_contractAddress;
-	u256 const m_gasPrice = 100 * szabo;
+	u256 const m_gasPrice = u256(100) << 12;
```

Comment:
Why not keep `gwei` above and set this to `10 * gwei` as that is a standard price nowadays.